revision_scripts/module_modification.py

```diff
@@ -88,6 +88,26 @@ def matchDim():
     return matchDim()(module.num_features)
 
 
+def _batchnorm_to_bn_without_stats(
+    module: nn.modules.batchnorm._BatchNorm, keep_running_stats: bool = True
```

Comment:
why is `keep_running_stats` here True?

Reply:
it was not used, but fixed in 1aa7f769d827df1313f247349b30fd6d3658cc18